src/scheduler/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@openintegrationhub/scheduler",
+  "description": "Scheduler framework",
+  "version": "0.0.1-dev.2",
+  "author": "Open Integration Hub",
+  "engines": {
+    "node": "^8.0.0"
+  },
+  "scripts": {
+    "pretest": "eslint ./src",
+    "test": "mocha spec --recursive"
+  },
+  "dependencies": {
+    "bunyan": "1.8.12"
+  },
+  "devDependencies": {
+    "chai": "^4.2.0",
```

Comment:
plz don't use ^ versions. This is possible source of problems

Reply:
@anton-kotenko fixed